drivers/vulkan/rendering_device_driver_vulkan.cpp

```diff
@@ -2681,6 +2752,14 @@ void RenderingDeviceDriverVulkan::_swap_chain_release(SwapChain *swap_chain) {
 	swap_chain->framebuffers.clear();
 
 	if (swap_chain->vk_swapchain != VK_NULL_HANDLE) {
+#if defined(SWAPPY_FRAME_PACING_ENABLED)
+		if (swappy_frame_pacer_enable) {
+			// Swappy has a bug where the ANativeWindow will be leaked if we call
```

Comment:
1. I already reported it to them, I haven't seen the fix yet. But it's been a while since the last time I checked their public source code.
2. It's still a good practice to do that, Swappy bug or not. Furthermore Swappy could end up "solving it" just change their API semantics to force the user to first destroy the Swapchain. The comment is basically explaining: Do NOT remove that call, Swappy won't do it for you (IMHO it's a Swappy bug nonetheless).